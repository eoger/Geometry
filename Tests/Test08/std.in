C Cercle1 5 5 5
C Cercle2 5 5 5
C Cercle3 5 5 5
LIST
DELETE CerclePasBon Cercle1 Cercle2 Cercle3
LIST
DELETE Cercle1 Cercle2
LIST
DELETE Cercle3
LIST
DELETE Cercle3
C Cercle 5 5 5
R Rect 1 2 3 4
L Ligne 1 2 3 4
PL PolyL 1 2 3 4 5 6
OA oa1 Cercle Ligne PolyL
OA oa2 oa1 PolyL Rect
OA oa3 oa1 oa2
LIST
DELETE PolyL
LIST
DELETE oa2
LIST
EXIT