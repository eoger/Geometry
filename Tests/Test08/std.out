OK
OK
OK
C Cercle1 5 5 5
C Cercle2 5 5 5
C Cercle3 5 5 5
ERR
C Cercle1 5 5 5
C Cercle2 5 5 5
C Cercle3 5 5 5
OK
C Cercle3 5 5 5
OK
ERR
OK
OK
OK
OK
OK
OK
OK
C Cercle 5 5 5
L Ligne 1 2 3 4
OA oa1 Cercle Ligne PolyL
OA oa2 PolyL Rect oa1
OA oa3 oa1 oa2
PL PolyL 1 2 3 4 5 6
R Rect 1 2 3 4
OK
C Cercle 5 5 5
L Ligne 1 2 3 4
OA oa1 Cercle Ligne
OA oa2 Rect oa1
OA oa3 oa1 oa2
R Rect 1 2 3 4
OK
C Cercle 5 5 5
L Ligne 1 2 3 4
OA oa1 Cercle Ligne
OA oa3 oa1
R Rect 1 2 3 4

